index.html

```diff
@@ -30,7 +30,7 @@ <h1>Hey There. <br>I’m Elijah</h1>
     </div>
 
       <h2 class="Descriptive">I’am a Software Developer</h2>
-      <p class="details">I can help you build a product feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p class="details">I can help you build a product feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestitate to contact me.</p>
     <section>
       <img class="pics" src="img/Group.png" alt="">
     </section> 
```

Comment:
You have done a great job here !, but I will suggest you should use 'div' to replace 'section' this will make your code looks more professional